client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -123,6 +124,12 @@ class EmailProvidersComparison extends React.Component {
 		this.isMounted = false;
 	}
 
+	getComparisonContext() {
+		const { comparisonContext = 'email-purchase' } = this.props;
+
+		return comparisonContext;
+	}
+
```

Comment:
This method looks a bit strange to me. I think it's possible to define a default value with `defaultProps`, no?

Reply:
🤦 I'll do that! I've gotten used to writing functional components instead of class-based components!